### PR TITLE
Add bss-init to initialize Postgres DB

### PR DESCRIPTION
**Pull request by synackd**
### Summary and Scope

Use the migration approach to database initialization. Similar to [SMD](https://github.com/OpenCHAMI/smd)'s [`smd-init`](https://github.com/OpenCHAMI/smd/tree/main/cmd/smd-init), add `bss-init`, which initializes a Postgres database using Golang's [migrate](https://github.com/golang-migrate/migrate) library. Convert the PostgreSQL queries that were originally in the `boot-script-services` binary to be migrations used by `bss-init`.

### Issues and Related PRs

* Addresses https://github.com/OpenCHAMI/roadmap/issues/20
* Addresses https://github.com/OpenCHAMI/bss/issues/5

### Testing

Tested on:

- Cable Guys SI Cluster (`cg-head` and compute nodes)
  - This is a system of x86-64 Gigabyte 272-Z32-00 machines running Rocky Linux 8.8

### CODE REVIEW

File: migrations/postgres/1_create_db_init.down.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+-- MIT License
+--
+-- (C) Copyright [2019-2021] Hewlett Packard Enterprise Development LP
```
Comment (davidallendj) [inline migrations/postgres/1_create_db_init.down.sql:3]: Should the copyright be changed to reflect the license used for OChami?

Comment (synackd) [reply migrations/postgres/1_create_db_init.down.sql:3]: Resolved in [80f702c](https://github.com/OpenCHAMI/bss/pull/13/commits/80f702ce021c50f3e44003ab18574dcf17bb9078).

**Review by davidallendj — APPROVED** [review]

**Comment by alexlovelltroy** [comment]
I don't see any changes to the .goreleaser.yaml file or the .github/workflows/actions.  Those need to be updated in order for this work to get automatically built and released.  If you're not comfortable doing that in this PR, I can open a fresh one to take care of it once this merges.

@synackd what do you think?

**Comment by synackd** [comment]
> I don't see any changes to the .goreleaser.yaml file or the .github/workflows/actions. Those need to be updated in order for this work to get automatically built and released. If you're not comfortable doing that in this PR, I can open a fresh one to take care of it once this merges.
> 
> @synackd what do you think?

Ah, I missed that! Thank you for pointing that out. Updated in [3bffbae](https://github.com/OpenCHAMI/bss/pull/13/commits/3bffbaef7d5aceb76dc3e451bfbd6e754ca4b59a).

**Comment by synackd** [comment]
@alexlovelltroy I realized that I didn't edit the `.github/workflow` files, which I did in above most recent commits. Can you check these changes?

**Comment by alexlovelltroy** [comment]
Running goreleaser locally, I discovered that you're missing migrations in the extra_files section under dockers.  I fixed it locally which made it run.  Here's the change you need:

```    extra_files:
      - LICENSE
      - CHANGELOG.md
      - README.md
      - .version
      - migrations/
   ```

**Comment by synackd** [comment]
Added in [c39fb13](https://github.com/OpenCHAMI/bss/pull/13/commits/c39fb13de98ab2205fbd94a0202124c166c15fdb).

**Review by alexlovelltroy — APPROVED** [review]
LGTM

File: go.mod
[the remark this answers is not in the document]
Comment (synackd) [reply go.mod:1]: Agreed, will be nice to clear those out in the future.

**Pull request merged**